doc/ikfast_tutorial.rst

```diff
@@ -38,26 +38,12 @@ OpenRAVE Installation
 ----------------------
 **Binary Install**
 
-OpenRave is available for Ubuntu up to version 12.04 using these commands::
+ sudo add-apt install ros-indigo-openrave
```

Comment:
- does ``add-apt`` work with 12.04 or only newer Ubuntu versions? seems we shouldn't switch yet
- can we change ``indigo`` to ``$ROS_DISTRO``?
- the first sentence should remain and read something like "OpenRave is available for Ubuntu up to version 14.04 using these commands:"